fs/fs.go

```diff
@@ -450,20 +459,31 @@ func getDiskStatsMap(diskStatsFile string) (map[string]DiskStats, error) {
 		}
 		// 8      50 sdd2 40 0 280 223 7 0 22 108 0 330 330
 		deviceName := path.Join("/dev", words[2])
+
+		var error error
+		devInfo := make([]uint64, 2)
+		for i := 0; i < len(devInfo); i++ {
```

Comment:
should we just remove the offset in the loop below instead of adding another loop here?

Reply:
@dashpole I actually thought of doing that but, a typical line from `/proc/diskstats` looks like this,

```
 253       2 dm-2 252099 0 2505842 36069 44352 0 856832 125528 0 25004 161597 0 0 0 0 0 0
```
Without this patch the code is offsetting first 3 elements (delimited by a space), to get the metrics. Now with this patch, we are also interested in getting first two elements (major number and minor number). 

If we try to remove the offset and parse all the element, the third element (name of the device) will fail at `strconv.ParseUint` call because it's not an integer. 

This is the reason we have one loop for first two elements which gathers disk information about major/minor number and the second loop (from 4th element onwards) which gathers disk statistics.

Reply:
Maybe we can keep the original loop, remove the offset and skip the 3rd element. But I am still not sure if we should store major/minor number in a slice that's named `stats`. IMHO, major/minor number has nothing to do with `stats`, they are more like device information. 

@dashpole WDYT?

Reply:
the second loop is fine.  Thanks for the explanation.